src/outfit.ts

```diff
@@ -446,6 +512,18 @@ export class Outfit {
       usedSlots.add(unusedSlot);
     }
 
+    // Handle the rider slots next
+    const bjorn = this.riders.get($slot`buddy-bjorn`);
+    if (bjorn) {
+      if (myEnthronedFamiliar() === bjorn) enthroneFamiliar($familiar.none);
+      if (myBjornedFamiliar() !== bjorn) bjornifyFamiliar(bjorn);
```

Comment:
Does this work if the bjornified familiar is the current familiar? Which can happen if e.g. this.familiar is not set, but the familiar to be bjornified was already out.

Reply:
Looks like if our current familiar is the one we want to bjornify, it will successfully get bjornified but leave us with Familiar.none

Reply:
Which is to say 'I have no idea what good behavior there is'

Reply:
That sounds fine. I view not specifying a familiar as a "don't care" for which equipping no familiar is a valid option.